lib/string_reverse.rb

```diff
@@ -1,4 +1,17 @@
 # A method to reverse a string in place.
+
 def string_reverse(my_string)
-  my_string << "not implemented"
+  if my_string == nil || my_string.length == 0
+    return my_string

```

Comment:
Minor: since you're reversing the string in place, you don't need to return my_string. Line 5 could simply be `return`